docs/analysis_plan.md

```diff
@@ -0,0 +1,90 @@
+# Analysis plan
+
+This document describes the analysis plan to perform a retrospective evaluation of the impact of incorporating wastewater data on the forecast performance of a semi-mechanistic renewal-based model.
+Forecasts of COVID-19 hospital admissions will be generated weekly and evaluated
+against the final observed hospital admissions data, across all age groups.
+
+# Aims
+
+The goal of these analyses will be to assess:
+1. How does incorporating wastewater impact the forecast performance of the semi-mechanistic renewal model?
+2. How do characteristics of the forecast problem and the wastewater surveillance in that forecast problem impact variation in performance between models with and without wastewater? 
```

Comment:
needs a reword/tighten to say forecast problem fewer times

Reply:
Revised